designs/developer/operator-virtualization-45/vm-sidepanel-resources.md

```diff
@@ -0,0 +1,8 @@
+---
+parent: Developer
+version: 4.5
+---
+
+## Design for the Resource tab for VM-based resources on topology
+
+![Design for vm-based resources resource tab](designs/developer/operator-virtualization-45/img/vm-topology-sidepanel-resource.png)
```

Comment:
```suggestion
![Design for vm-based resources resource tab](img/vm-topology-sidepanel-resource.png)
```

Reply:
Image wasn't loading for me but Beau's suggestion looks like it works okay...added as a github suggestion to make it easy for you to just apply :)